new_mrp/__manifest__.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+{
+    'name': 'New Mrp',
+    'version': '1.0',
+    'description': 'Custom MO Delivery Note report without Kit Products(in Inventory)',
+    'depends': ['mrp', 'stock' , 'sale_management'],
```

Comment:
```suggestion
    'depends': ['mrp', 'sale_management'],
```
stock is already installed if mrp is installed as stock is in depends of mrp module.

Reply:
okay noted!!